pkgs/development/ocaml-modules/ocolor/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchurl
+, buildDunePackage
+, cppo
+}:
+
+buildDunePackage rec {
+  pname = "ocolor";
+  version = "1.3.1";
+
+  minimalOCamlVersion = "4.02";
+
+  src = fetchurl {
```

Comment:
Please use `fetchzip` or `fetchFromGitHub`.

Reply:
Is this good?